website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'typescript-eslint-parser/package.json';
+import SettingsRenderer from '../utils/SettingsRenderer';
+
+const ID = 'typescript-eslint-parser';
+
+const defaultOptions = {
+  range: true,
+  loc: false,
+  tokens: false,
+  comment: false,
+  tolerant: true,
```

Comment:
Perhaps should be `false` by default? (so that user would see proper syntax errors)